src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -20,16 +24,57 @@ const BudgetDetailActivityTabContents = ({
     fetchOfferRedemptions,
   } = useOfferRedemptions(enterpriseUUID, enterpriseOfferId, subsidyAccessPolicyId);
 
-  return (
-    <LearnerCreditAllocationTable
-      isLoading={isLoadingOfferRedemptions}
-      tableData={offerRedemptions}
-      fetchTableData={fetchOfferRedemptions}
-      enterpriseUUID={enterpriseUUID}
-      enterpriseSlug={enterpriseSlug}
-      enableLearnerPortal={enableLearnerPortal}
-    />
-  );
+  if (!isTopDownAssignmentRealTimeLcmEnabled) {
+    return (
+      <LearnerCreditAllocationTable
+        isLoading={isLoadingOfferRedemptions}
+        tableData={offerRedemptions}
+        fetchTableData={fetchOfferRedemptions}
+        enterpriseUUID={enterpriseUUID}
+        enterpriseSlug={enterpriseSlug}
+        enableLearnerPortal={enableLearnerPortal}
+      />
+    );
+  }
+  if (!hasPendingAssignments && !hasCompletedTransactions) {
+    return <NoBudgetActivityCard />;
+  }
+  if (!hasPendingAssignments && hasCompletedTransactions) {
+    return (
+      <>
+        <NoBudgetActivityCard />
```

Comment:
[clarification] Is this roughly where the more recent empty state variation to handle the case of "no remaining assignments in the table because there are completed transactions" would go?

<img width="639" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/3c0e0773-2aee-4251-b8af-de3b9fda6a61">